src/Agent/NewRelic/Agent/Core/Configuration/DefaultConfiguration.cs

```diff
@@ -1051,6 +1051,15 @@ private void GetInfiniteTracingFlakyAndDelayTestSettings()
                 _infiniteTracingObserverTestFlaky = TryGetAppSettingAsFloat("InfiniteTracingSpanEventsTestFlaky");
             }
 
+            if (int.TryParse(_environment.GetEnvironmentVariable("NEW_RELIC_INFINITE_TRACING_SPAN_EVENTS_TEST_FLAKY_CODE"), out var flakyCodeVal))
```

Comment:
Maybe not for this PR, but we should consider adding the NEW_RELIC_INFINITE_TRACING_... env vars to our [docs ](https://docs.newrelic.com/docs/agents/net-agent/configuration/net-agent-configuration)site.
![image](https://user-images.githubusercontent.com/20267975/105771612-aa3cdf00-5f15-11eb-9272-91d1914664cf.png)

Reply:
That's a good idea.  However, we should exclude the env vars that configure the "flaky" test settings because those are only for internal testing.

Some of the configurable settings, like the span batch size, queue size and partition size, probably need to be explained in  enough detail so that customers can understand why they would want to change these settings (or leave them alone).